supplier_app/tests_company.py

```diff
@@ -0,0 +1,122 @@
+from django.test import (
+    Client,
+    TestCase,
+)
+
+from django.urls import (
+    reverse_lazy
+)
+
+from supplier_app.models import (
+    Company,
+    TaxPayer,
+    CompanyUserPermission,
+)
+from supplier_app.views import (
+    CompanySelectorView,
+)
+from users_app.models import User
+
+TAXPAYER = {
+    'name': 'Eventbrite',
+    'workday_id': '12345',
+}
+COMPANY = {
+    'name': 'Eventbrite',
+    'description': 'Bringing the world together through live experiences',
+}
+USER = {
+    'email': 'pepe@gmail.com',
+    'password': '123456'
+}
```

Comment:
We try to avoid the use of global variables. There are some options to do this, you can move it to the setUp of each test class; or you can create a super class and put it in as attributes, so later you can get them on each class in the inheritance; or you can create a factory class.